awx/api/serializers.py

```diff
@@ -3204,7 +3204,7 @@ def build_standard_field(self, field_name, model_field):
             field_kwargs['choices'] = module_name_choices
             field_kwargs['required'] = bool(not module_name_default)
             field_kwargs['default'] = module_name_default or serializers.empty
```

Comment:
For similar reasons as below, I'm guessing this maybe should just be `field_kwargs['default'] = module_name_default` ?

Reply:
Based on what I see, I believe the point of the original machinations here is for the `command` module to be the implicit default, if it is in the choices list provided by `settings.AD_HOC_COMMANDS`.